src/decimal.rs

```diff
@@ -112,6 +112,7 @@ pub struct UnpackedDecimal {
     archive_attr(derive(Clone, Copy, Debug))
 )]
 #[cfg_attr(feature = "rkyv-safe", archive(check_bytes))]
+#[cfg_attr(feature = "bitcode", derive(bitcode::Encode, bitcode::Decode))]
```

Comment:
Should also consider implementing manually to save 3 bytes per `Decimal`, or a ~18.75% size improvement. In particular, can store `flags` in `u8` instead of `u32` (because it only has 7 meaningful bits).